sw/device/silicon_creator/lib/drivers/alert.h

```diff
@@ -0,0 +1,105 @@
+#ifndef LOCAL_GOOGLE_HOME_CFRANTZ_OPENTITAN_OPENTITAN_SW_DEVICE_SILICON_CREATOR_LIB_DRIVERS_ALERT_H_
```

Comment:
Someone forgot to run the include handler fixer. =P

Reply:
I forgot all the things for this PR.  I've run the include-fixer, clang-format and added the needed copyright headers.